src/misc/windowing.jl

```diff
@@ -74,6 +72,38 @@ function windowmap!(f::Function, out::AbstractVector, x::AbstractVector; kwargs.
     return out
 end
 
+"""
+    slidebracket(t::AbstractVector, bracketing::Symbol; kwargs...)
```

Comment:
ok I'm still not convinced about the existence of this function. What was wrong with using `windowmap` with the functions `first`, `last` or `midpoint` as I had created before? I don't see any decrease of complexity here. You use one function with one argument "bracketing". Before we also used one function with one argument, but the argument was a function intead of a symbol. 

The previous version is better. First, it decreases the learning curve because we re-use the same `windowmap` function. Second, it is flexible: any function can be used, instead of we having to essentially translate a symbol into a code via an if clause.

Reply:
Ok, I had introduced this because we need the time vector of the indicators and the change metrics to precompute some of them. I agree with your point though: no need to introduce new keywords when we already have functions that do this. I think it would be nice if the user just does something like:

```julia
t = ...
indconfig = ...
t_ind = windowmap(midpoint, t, indconfig)
```

With an additional method for `windowmap` that safely passes the same windowing parameters as the one in `indconfig`.